src/exercise/04.html

```diff
@@ -6,17 +6,30 @@
   <script src="https://unpkg.com/react@17.0.0/umd/react.development.js"></script>
   <script src="https://unpkg.com/react-dom@17.0.0/umd/react-dom.development.js"></script>
   <script src="https://unpkg.com/@babel/standalone@7.12.4/babel.js"></script>
+  <script src="https://unpkg.com/prop-types@15.7.2/prop-types.js"></script>
   <script type="text/babel">
     // 🐨 Make a function called `message` which returns the JSX we want to share
+    const Message = ({ greeting, subject }) => <div className="message">{greeting}, {subject}</div>;
+
+    Message.propTypes = {
+      greeting: PropTypes.string.isRequired,
+      subject: PropTypes.string.isRequired,
+    };
 
     // 🐨 use that function in place of the divs below with:
     // 💰 {message({children: 'Hello World'})} {message({children: 'Goodbye World'})}
-    const element = (
+   /* const element = (
       <div className="container">
         <div className="message">Hello World</div>
         <div className="message">Goodbye World</div>
       </div>
-    )
+    )*/
+    const element = (
+        <React.Fragment className="container">
```

Comment:
since Fragments don't render any html elements, the `className` won't do anything here...

Reply:
also you can just use `<>` instead of `<React.Fragment>`